frontend/src/routes/manage/Realm/AddChild.tsx

```diff
@@ -68,6 +69,7 @@ const query = graphql`
             isRoot
             path
             canCurrentUserEdit
+            ancestors { name path }
```

Comment:
Seeing all these changes to the queries gathered like this I wonder whether it wouldn't be more idiomatic for `Breadcrumbs` to just expose a fragment and take a fragment ref. :thinking: 

And there are other things; in general, using `Breadcrumbs` on a page currently seems to come with a lot of boilerplate.

Reply:
Yeah, I noticed as well, but I'm not sure if we can easily fix it. Sure `BreadcrumbsFromRealm` would be possible, but most cases we want to have a special last segment (video title, "edit", ..). So that needs to be configurable again? So my quick estimate is that we wouldn't really save that much duplicate code and add a lot of new code. But I'm not sure. 

Would you like me to try to refactor stuff in this PR still?

Reply:
Good point. So maybe not in this PR anymore, no. But something to keep in mind. :shrug: